### ISSUE TITLE
Details of the loss function during training

### ISSUE THREAD

**Issue by woshixyr** [LianpoWang/GMPTV#19]
The code mentions fluidflow_loss_func but does not elaborate on its composition. Does this loss function incorporate endpoint error (EPE) or other geometric constraints? When training different types of flow field data, is it necessary to adjust the weights of the loss function?
